ZeeHomeScreen/Classes/Components/BasicContainerCell/UniversalCollectionViewCell.h

```diff
@@ -5,30 +5,15 @@
 //  Created by Miri on 28/01/2020.
 //
 
+@import UIKit;
 
-@import ZappPlugins;
-#import <Foundation/Foundation.h>
-#import "ComponentDelegate.h"
-@class ComponentModel;
-@class ComponentProtocol;
+@protocol ComponentProtocol;
 
-/**
- *  This cell used as container for all components. Do not remove or modify anything without premission
- */
 @interface UniversalCollectionViewCell: UICollectionViewCell
 
-@property (nullable, nonatomic, strong) UIViewController<ComponentProtocol> *componentViewController;
+@property (nonatomic, weak) UIViewController<ComponentProtocol> *componentViewController;
+@property (nonatomic) BOOL parentHandlesReusingComponent;
```

Comment:
Introduced a new property.
By default, cell just removes it's hosted controller when reused. Parent might set this property to YES, and handle reusing as wished. Improves performance a little bit